### DISCUSSION TITLE
Is CREATE NOTEBOOK privileges supported?

### DISCUSSION THREAD

**Discussion by anitalySuper** [Snowflake-Labs/terraform-provider-snowflake#2770] [Q&A]
Snowflake has a new notebook feature in private preview: https://docs.snowflake.com/LIMITEDACCESS/snowsight-notebooks/ui-snowsight-notebooks-use#label-notebooks-required-privileges.

Is the privilege CREATE NOTEBOOK on schema supported?

**Comment by sfc-gh-jcieslak** [comment]
Hey 👋, If you're only asking about the CREATE NOTEBOOK privilege, I'll add it during some of the other work and let you know, but next time please create a GitHub issue with the feature request template.

**Reply by anitalySuper** [reply]
Thanks, I was more so wondering if there is a list of supported privileges I can refer to.

**Reply by sfc-gh-jcieslak** [reply]
There's no such list, but if you can, please create a gh issue to add it to the documentation (you can refer to this thread). The list should be publicly available so everyone can compare it to the Snowflake's available privilege list. Also, I'm working on adding missing privileges, so CREATE NOTEBOOK will be available in the next release (probably today).

**Reply by sfc-gh-jcieslak** [reply]
Hi again 👋
Today we released a new version v0.90.0. Please, let us know if you're able to grant CREATE NOTEBOOK after migrating (also mind there were small breaking changes documented [here](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/MIGRATION_GUIDE.md#v0890--v0900)).
